frontend/tests/unit/components/layout/Tooltip.spec.ts

```diff
@@ -5,7 +5,7 @@ import Tooltip from '@/components/layout/Tooltip.vue';
 async function createWrapper(options?: {
   hint?: string;
   position?: string;
```

Comment:
Can we remove all the occurences of `position` in the test file since we removed it from the props definition of the component?

Reply:
Somehow missed some...

Reply:
:heavy_check_mark: